taca/deliver/cli.py

```diff
@@ -0,0 +1,100 @@
+""" CLI for the deliver subcommand
+"""
+import click
+from ngi_pipeline.utils.communication import mail_analysis
```

Comment:
@b97pla : Can we pull the `util` and add in `TACA`'s own util. Just to avoid dependency on `ngi_pipeline`. Thats what @guillermo-carrasco and @vezzi  suggested me as well :) 

But I notice there other `utils` and `database` object are imported as well, so instead of pulling out maybe can add `ngi_pipeline` to `requirements`. Or even better if `ngi_pipeline.database` is mainly focused for `Charon` cant we put them in a separate repo as [statusdb](https://github.com/SciLifeLab/statusdb). Than it cant made as requirement for packages like `ngi_pipeline, taca, ngi_reports`.